src/ethereum_test_tools/code/yul.py

```diff
@@ -4,16 +4,34 @@
 
 from pathlib import Path
 from subprocess import PIPE, run
-from typing import Optional
+from typing import Mapping, Optional, Tuple, Type
+
+from ethereum_test_forks import Fork
 
 from .code import Code
 
 SOLC: Path = Path("solc")
-SOLC_ARGS = (
-    SOLC,
-    "--assemble",
-    "-",
-)
+DEFAULT_SOLC_ARGS = ("--assemble", "-")
+
+
+def get_evm_version_from_fork(fork: Fork | None):
+    """
+    Get the solc evm version corresponding to `fork`.
+
+    Args
+    ----
+        fork (Fork): The fork to retrieve the corresponding evm version for.
+
+    Returns
+    -------
+        str: The name of evm version as required by solc's --evm-version.
+    """
+    if not fork:
+        return None
+    fork_to_evm_version_map: Mapping[str, str] = {"Merge": "paris"}
```

Comment:
Happy to do that (in a separate PR) :) we'd have to patch it on geth's evm tool though, because in the help it's known as "Merge" :upside_down_face: :

```
evm t8n --help
```

![grafik](https://github.com/ethereum/execution-spec-tests/assets/91727015/54c9ec3f-a696-4ad5-af79-d19d6332910c)